packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -279,7 +280,12 @@ export function publish(options: InitOptions): void {
         flowletName += `${separator}element=${autoLoggingID}`;
       }
       flowletName += ')';
-      let callFlowlet = new flowletManager.flowletCtor(flowletName, ALUIEventGroupPublisher.getGroupRootFlowlet(event));
+
+      let triggerFlowlet = new flowletManager.flowletCtor(flowletName, ALUIEventGroupPublisher.getGroupRootFlowlet(event));
+      let callFlowlet = Flags.getFlags().preciseTriggerFlowlet
```

Comment:
Should the check be the inverse, i.e. `!Flags.getFlags().preciseTriggerFlowlet`?

Reply:
I don't think so. To test and toll out the new logic I assume we set the flag to true. The "new" logic is to read `.top()` of  the stack. Otherwise, the fallback to old behavior is that callflowlet and triggerflowet will be the same. 
Note that I renamed the old `callFlowlet` to`triggerFlowlet` since that was the actual object.

Reply:
Ah makes sense, sorry I got confused